### PR TITLE
adding addition default collections on namespaces

### PR DESCRIPTION
**Pull request by sferich888**
and defaults for --defaults

### CODE REVIEW

File: pkg/cmd/inspect/inspect.go [hunk 1]
```diff
@@ -41,6 +41,8 @@ var (
 var (
 	defaults = []string{
 		"clusteroperator",
+		"nodes",
+		"persistentvolumes",
```
Comment (sferich888) [inline pkg/cmd/inspect/inspect.go]: I don't think this will work due to how args are read currently.

Comment (juanvallejo) [reply pkg/cmd/inspect/inspect.go]: it'll work as long as any args provided by the user aren't resource/name pairs

Comment (juanvallejo) [reply pkg/cmd/inspect/inspect.go]: cc @deads2k ^

Comment (sferich888) [reply pkg/cmd/inspect/inspect.go]: We can't assume that no one does`mg inspect --defaults namespace abc` well have to find a way to mitigate this.

Comment (juanvallejo) [reply pkg/cmd/inspect/inspect.go]: We could think about making  arguments and `--defaults` mutually exclusive

**Comment by sferich888** [comment]
Wasn't that the intent, to begin with?

File: pkg/cmd/inspect/inspect.go (hunk 1 above)
Comment (davemulford) [reply pkg/cmd/inspect/inspect.go]: Why would both not work?  Seems like we should be able to gather both the defaults, and whatever else the user adds on?

**Comment by deads2k** [comment]
we decided against this direction.

**Pull request closed without merging**